pkg/config/model/viper.go

```diff
@@ -670,6 +693,12 @@ func (c *safeConfig) AddConfigPath(in string) {
 	c.Viper.AddConfigPath(in)
 }
 
+func (c *safeConfig) AddExtraConfigPaths(in []string) {
+	c.Lock()
+	defer c.Unlock()
+	c.extraConfigFilePaths = append(c.extraConfigFilePaths, in...)
+}
+
```

Comment:
This method would have no impact after `ReadInConfig` is called leading to unexpected behavior. We should refuse any new file after `ReadInConfig` is called or take the list of extra file as parameter to `ReadInConfig`.

Reply:
We could have the same argument for `AddConfigPath`, it seems that it's the way Viper work:
1. you add a config file to the Viper instance 
2. you load it with `ReadInConfig()`

I make sure that calling `ReadInConfig()` function several time in a row ends with the same result as if it was called only one time.

Reply:
It's true that we have this problem everywhere with viper. Let's fix it in a later PR.